covsirphy/phase/sr_change.py

```diff
@@ -129,8 +129,17 @@ def _curve_fitting(self, phase, start_date, end_date):
         sr_df = sr_df.loc[(sr_df.index >= sta) & (sr_df.index <= end), :]
         trend = Trend(sr_df)
         df = trend.run()
+        if trend.rmsle() > trend.max_linear_fit_err:
```

Comment:
Please consider to perform both of linear regression and curve fitting with exponential function. Then, select better function using RMSLE score.

```Python
# Linear function
trend_line = Trend(sr_df)
line_df = trend_line.run(func="linear")
# Negative exponential function
trend_exp = Trend(sr_df)
exp_df = trend_exp.run(func="negative_exppnential")
# Compare the trends with RMSLE score
df = line_df.copy() if trend_line.rmsle() < trend_exp.rmsle() else exp_df.copy()
```